components/molecules/MainThumbnailList/MainThumbnailList.tsx

```diff
@@ -29,15 +29,19 @@ export function MainThumbnailList({
 }) {
   return (
     <S.MainThumbnailListContainer>
-      {postList.map(({ postId, postThumbnailPath, title, createDate }) => (
-        <ThumbnailCard
-          key={postId}
-          postId={postId}
-          postThumbnailPath={postThumbnailPath}
-          title={title}
-          createDate={createDate}
-        />
-      ))}
+      {postList.length ? (
+        postList.map(({ postId, postThumbnailPath, title, createDate }) => (
+          <ThumbnailCard
+            key={postId}
+            postId={postId}
+            postThumbnailPath={postThumbnailPath}
+            title={title}
+            createDate={createDate}
+          />
+        ))
+      ) : (
+        <div>검색된 결과가 없습니다</div>
+      )}
```

Comment:
삼항 연산자 보다 &&를 사용하는 것이 명확해보여 저는 보통 이렇게 사용하는것을 좋아합니다 :) ( 언제까지나 취향 )

```suggestion
      {postList.length &&
        postList.map(({ postId, postThumbnailPath, title, createDate }) => (
          <ThumbnailCard
            key={postId}
            postId={postId}
            postThumbnailPath={postThumbnailPath}
            title={title}
            createDate={createDate}
          />
        ))}
      {!postList.length && <div>검색된 결과가 없습니다</div>}
```

Reply:
취존합니다